server/dumpbagserver/templates/dumps.html

```diff
@@ -63,9 +60,22 @@ <h2 class="dump-title" onclick="toggleGroup('dump-list-for-{{database}}', '{{dat
   if (hash) {
     toggleVisibility('dump-list-for-' + hash);
   }
+  function search (text) {
+      var items = document.getElementsByTagName('li');
+      for (var i = 0; i < items.length; i++) {
```

Comment:
in order to hide the database names when no database is found:

* hide all the `<h2 class="dump-title"/>` before the for loop
* inside the for loop, when we find a dump matching the search, show the parent `<h2 class="dump-title"/>`

```javascript
        if (name.toUpperCase().indexOf(text.toUpperCase()) >= 0) {
              items[i].style.display = 'list-item';
              // here, find the parent h2.dump-title and show it
         }
```